plugins/processors/dcos_metadata/README.md

```diff
@@ -31,9 +37,11 @@ This process adds the following tags to any metric with a container_id tag set:
  - `service_name` - the name of the service (mesos framework) which scheduled 
                     the task associated with this container
 
-Additionally, any task labels which are prefixed with `DCOS_METRICS_` are added
-to each metric as a tag. For example, the application configuration would have
-every metric associated with it decorated with a `FOO=bar` tag.
+Additionally, any task labels which are prefixed with strings included in the configurable whitelist of prefixes
+(`whitelist_prefix`) are added to each metric as a tag. For example, the application configuration would have every
+metric associated with it decorated with a `FOO=bar` tag if `whitelist_prefix` was configured to include
+`DCOS_METRICS_`. Any task labels that are included in the configurable whitelist (`whitelist`) are also added to each
```

Comment:
Note that this is a breaking change for the current 1.12 release as `DCOS_METRICS_` is not longer included by default.

Reply:
will include in dcos changelog